stores/amazon.py

```diff
@@ -132,10 +134,36 @@ def something_in_stock(self):
         f.set(params)
         self.driver.get(f.url)
         selenium_utils.wait_for_any_title(self.driver, ADD_TO_CART_TITLES)
-        if self.driver.find_elements_by_xpath('//td[@class="price item-row"]'):
+        prices = self.driver.find_elements_by_xpath('//td[@class="price item-row"]')
+        prices2 = self.driver.find_elements_by_xpath('//td[@class="price item-row"]')
+        dont_refresh = True
+
+        if prices:
             log.info("One or more items in stock!")
 
-            return True
+            # Check against limit price
+            for price_str in prices:
+                price_int = int(round(float(price_str.text.strip("$"))))
+
+                if price_int <= self.price_limit:
```

Comment:
When the price is too high, it just continues to refresh without sending any log statement. Lets atleast log that the item is in stock but the price limit is too high.

Reply:
Added and fixed prices with commas.  Not sure if amazon stores in other locales format numbers differently.